core/templates/dev/head/filters.js

```diff
@@ -144,9 +144,7 @@ oppia.filter('isOutcomeConfusing', [function() {
   return function(outcome, currentStateName) {
     return (
       outcome.dest === currentStateName &&
-      !outcome.feedback.some(function(feedbackItem) {
-        return feedbackItem.trim().length > 0;
-      })
+      !outcome.feedback.getHtml().trim().length > 0
```

Comment:
I wonder if we could get rid of this filter altogether and just do outcome.isConfusing()?

Reply:
done